client/src/pages/mocks/MockResponseCard.tsx

```diff
@@ -126,7 +128,18 @@ export const MockResponseCard: React.FC<IMockResponseCard> = ({
             </div>
           </div>
           <div className="flex flex-row items-center space-x-2">
-            {isDeleting ? (
+
+         {   onDuplicateMockResponse&&  <Tooltip
+                title="Duplicate mock response"
+              >
+                <FileCopy sx={{fontSize: '15px' }}
```

Comment:
The copy icon isn't accurate for duplicate.
Try to use one of those:
https://react-icons.github.io/react-icons/search/#q=duplicate

Reply:
ok